commands/publish/README.md

```diff
@@ -62,6 +63,15 @@ This is useful when a previous `lerna publish` failed to publish all packages to
 - [`--yes`](#--yes)
 - [`--tag-version-prefix`](#--tag-version-prefix)
 
+### `--_auth`
+
+```sh
+lerna publish --_auth aGk6bW9t
+```
+
+When publishing packages that require authentication but you are working with an internally hosted NPM Registry that only uses the legacy Base64 version of username:password.
```

Comment:
This is probably not always true, but since NPM no longer advocates this approach, I thought it was ok to say.